src/main/java/com/google/sps/data/Meal.java

```diff
@@ -50,4 +51,28 @@ public String getDescription() {
     public String getType() {
         return this.type;
     }
+
+    /**
+     * Counts the number of keywords that are matching in recipe.
+     * @param params list of strings with keywords, that describe search request.
+     * @return amount of occurrences of keywords from params in this meal.
+     */
+    public int getFrequency(List<String> params) {
```

Comment:
I'd suggest adding a unit test for this method in a MealTest.java file.
A test that would check the exact number returned by this method for some instances.